server/models.py

```diff
@@ -285,6 +285,13 @@ class Assignment(Model):
     files = db.Column(JsonBlob)  # JSON object mapping filenames to contents
     course = db.relationship("Course", backref="assignments")
 
+    publish_tags = [tag.title() if tag != 'private' else 'Hidden' for tag in GRADE_TAGS]
+    grade_visibility = db.Column(db.String(255), nullable=False, default="Hidden")
```

Comment:
If we use `StringList` here, it shouldn't be nullable (the we have to deal with both null and the empty list). If we set the default value to the empty list the migration should go smoothly:

`db.Column(StringList, nullable=False, default=[])`